app/components/user-cards.js

```diff
@@ -31,6 +36,18 @@ export default Component.extend({
             return allteamlistArray.sort(
                 (item1, item2) => (get(item1,sortoption) < get(item2,sortoption)) ? 1 : (get(item1,sortoption) > get(item2,sortoption)) ? -1 : 0);
             }
+        }}
+        else {
+            return allteamlistArray.filter((element)=> {
+                if((get(element,'fullName')).toLowerCase().startsWith(this.displaysearch.toLowerCase())  || 
+                (get(element,'last_name')).toLowerCase().startsWith(this.displaysearch.toLowerCase()) ){
+                   return element;
```

Comment:
This is good. But I guess we are not supporting sort in search results. Please try to do the sort in the search results too.

Reply:
added sort functionality in search logic.